src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for this action type 👍